diff --git a/node_modules/node-pty/src/win/conpty.cc b/node_modules/node-pty/src/win/conpty.cc
index 47af75c..884d542 100644
--- a/node_modules/node-pty/src/win/conpty.cc
+++ b/node_modules/node-pty/src/win/conpty.cc
@@ -472,10 +472,6 @@ static NAN_METHOD(PtyKill) {
       }
     }
 
-    DisconnectNamedPipe(handle->hIn);
-    DisconnectNamedPipe(handle->hOut);
-    CloseHandle(handle->hIn);
-    CloseHandle(handle->hOut);
     CloseHandle(handle->hShell);
   }
 
